src/Eval.hs

```diff
@@ -315,13 +315,41 @@ eval ctx xobj@(XObj o info ty) preference =
               case res of
                 Right xobj' -> macroExpand ctx' xobj'
                 Left _ -> pure (ctx, res)
-        XObj (Lst [XObj (Command callback) _ _, _, _]) _ _ : args ->
+        [XObj (Lst [XObj (Command (NullaryCommandFunction nullary)) _ _, _, _]) _ _] ->
```

Comment:
I guess I should pattern-match to catch wrong number of arguments in all of these.